### PR TITLE
Update the documentation for the gateway related to HTTP API.

### PR DESCRIPTION
**Pull request by lee-emqx**

### CODE REVIEW

File: docs/en_US/gateway/dds.md [hunk 1]
```diff
@@ -162,6 +162,73 @@ Key configuration items include:
...
+	# # username
+	# #
+    # # Basic authorization 
+    # #
+	# # Value: String
+	username = admin
+	# # password
+	# #
+    # # Basic authorization
+    # #
```
Comment (wanghaEMQ) [inline docs/en_US/gateway/dds.md:191]: Duplicated description
```
    # # Basic authorization
    # #
```

File: docs/en_US/gateway/dds.md [hunk 2]
```diff
@@ -162,6 +162,73 @@ Key configuration items include:
...
+```shell
+$ curl --basic -u admin:public 'http://127.0.0.1:8082/api/v4/proxy/ctrl/stop' \
+--header 'Content-Type: application/json' \
+--data '{
+    "req": 10,
+    "action": "stop",
+    "seq": 1234
+}'
+```
+- 重启网关：
```
Comment (wanghaEMQ) [inline docs/en_US/gateway/dds.md:219]: Typo

File: docs/en_US/gateway/dds.md [hunk 3]
```diff
@@ -162,6 +162,73 @@ Key configuration items include:
...
+```
+
+- Update configuration file:
+```shell
+$ curl --basic -u admin:public 'http://127.0.0.1:8082/api/v4/proxy/configuration/dds' --header 'Content-Type: text/plain'  --data-binary '@nanomq_dds_gateway.conf'
+```
+
+- Restart gateway:
+```shell
+$ curl --basic -u admin:public 'http://127.0.0.1:8082/api/v4/proxy/ctrl/stop' \
```
Comment (wanghaEMQ) [inline docs/en_US/gateway/dds.md:211]: stop?

File: docs/en_US/gateway/someip-gateway.md [hunk 4]
```diff
@@ -106,14 +106,72 @@ gateway.vsomeip {
...
+	# # username
+	# #
+    # # Basic authorization 
+    # #
+	# # Value: String
 	username = admin
+	# # password
+	# #
+    # # Basic authorization
+    # #
```
Comment (wanghaEMQ) [inline docs/en_US/gateway/someip-gateway.md:136]: Duplicated description

**Review by wanghaEMQ — CHANGES_REQUESTED** [review]

File: docs/en_US/gateway/dds.md (hunk 3 above)
Comment (lee-emqx) [reply docs/en_US/gateway/dds.md:211]: Yes, it's.

File: docs/en_US/gateway/dds.md (hunk 2 above)
Comment (lee-emqx) [reply docs/en_US/gateway/dds.md:219]: Thank you, the issue has been resolved!

**Review by LenaLenaPan — APPROVED** [review]

**Review by wanghaEMQ — APPROVED** [review]

**Pull request merged**